src/middlewares/validationHandler/guestValidator.js

```diff
@@ -0,0 +1,25 @@
+import { check, validationResult } from "express-validator";
+
+exports.guestValidator = [
+    check('name').trim().not().isEmpty()
+    .withMessage("Please provide your Name")
+    .isLength({min: 5, max: 50})
+    .withMessage("Name must be within 5 to 50 Characters"),
+    check('email').normalizeEmail().isEmail()
+    .withMessage("Please provide a valid Email"),
+    check('phone').trim().not().isEmpty()
+    .withMessage("Please provide your Phone Number")
```

Comment:
Are you checking if the phone number and email already exist, to prevent duplicate records

Reply:
I have validated Phone Number and Email to prevent duplicate records.